syntax = "proto3";

package mcproto;

message DiskInfo
{
    uint64 availableSpace = 1;
}
